src/control/jobs/CustomExportJob.cpp

```diff
@@ -189,7 +186,7 @@ void CustomExportJob::exportPngPage(int pageId, int id, double zoom, DocumentVie
 		PdfView::drawPage(NULL, popplerPage, cr, zoom, page->getWidth(), page->getHeight());
 	}
 
-	if (this->chosenFilterName == EXPORT_PNG_NOBG)
+	if (this->chosenFilterName == _("PNG with transparent background"))
```

Comment:
It would be nicer to add the values to the map, so for value use e.g. 

class  ExportType {
public:
  string extension;
  bool transparentBg
}

or something like this. Because if the texts are updated on one place, and somebody don't know he has to update it on a second place, it will fail.